libdnf5/transaction/transaction_sr.hpp

```diff
@@ -49,6 +49,7 @@ struct PackageReplay {
     TransactionItemAction action;
     TransactionItemReason reason;
     std::string group_id;
+    // This nevra doesn't contain epoch if it is 0
     std::string nevra;
     std::filesystem::path package_path;
```

Comment:
May I ask you for a doc string for a package path?

Reply:
Added.